nix/haskell-project.nix

```diff
@@ -0,0 +1,102 @@
+{ inputs # Desystemized merged inputs 
+, systemized-inputs # Non-desystemized merged inputs
+, flakeopts # iogx config passed to mkFlake
+, pkgs # Desystemized nixpkgs (NEVER use systemized-inputs.nixpkgs.legacyPackages!)
+, ghc # Current compiler
+, deferPluginErrors # For Haddock generation
+, enableProfiling
+}:
+
+let
+  lib = pkgs.lib;
+  isDarwin = pkgs.stdenv.hostPlatform.isDarwin;
+  isCross = pkgs.stdenv.hostPlatform != pkgs.stdenv.buildPlatform;
+
+  antaeus-module = { config, ... }: {
+    packages = {
+      # Things that need plutus-tx-plugin
+      freer-extras.package.buildable = !isCross;
+      cardano-node-emulator.package.buildable = !isCross;
```

Comment:
may not need:
- `cardano-node-emulator`
- `cardano-streaming`
- `plutus-pab-executables`